src/SampleWebApp/Startup.cs

```diff
@@ -41,6 +42,11 @@ public Startup(IHostingEnvironment env, IApplicationEnvironment appEnv)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddDbContext<BookDataContext>(options =>
+                    options.UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=BookDataContext-36f2c634-aa7b-4dbd-80e6-95e06e3521a3;Trusted_Connection=True;MultipleActiveResultSets=true"));
```

Comment:
In this case yes, the AuthorssApiController and BooksApiController were generated as part of two separate scaffolds. In each I provided a new DataContext. If user input provides an existing DataContext, scaffolding will just try to add the model to the existing one. 

We do have an open issue for moving the connection string to appsettings.json, but won't be targeted for RC2.